core/utils.py

```diff
@@ -160,36 +160,42 @@ def create_embed(
         *,
         error: bool = False,
         author: Union[discord.Member, discord.User] = None,
-        **kwargs,
+        title: Union[str, discord.Colour] = discord.Embed.Empty,
+        description: Union[str, discord.Colour] = discord.Embed.Empty,
```

Comment:
`Union[str, discord.Embed.Empty]` instead of `Union[str, discord.Colour]` :)

Reply:
Oops